frontend/src/tournaments/LeaderboardTab.tsx

```diff
@@ -102,10 +106,28 @@ const LeaderboardTab = () => {
                         select
                         label='Site'
                         value={site}
+                        color='primary'
                         onChange={(e) => setSite(e.target.value as LeaderboardSite)}
                     >
-                        <MenuItem value={LeaderboardSite.Lichess}>Lichess</MenuItem>
-                        <MenuItem value={LeaderboardSite.Chesscom}>Chess.com</MenuItem>
+                        <MenuItem value={LeaderboardSite.Lichess}>
+                            {' '}
```

Comment:
I don't think we need this leading space

Reply:
ok got it